ToDoListAPI/ToDoListAPI/Controllers/TaskController.cs

```diff
@@ -0,0 +1,98 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using ToDoListAPI.IService;
+using ToDoListAPI.Models;
+using ToDoListAPI.Services;
+
+namespace ToDoListAPI.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class TaskController : ControllerBase
+    {
+        private ITaskServices taskService;
+        public TaskController(ITaskServices taskService)
+        {
+            this.taskService = taskService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetAllTask()
```

Comment:
```suggestion
        public async Task<ActionResult<....>> GetAllTask()
```
dùng ntn thì trên swagger mới có nhé